glam/api/views.py

```diff
@@ -219,12 +221,18 @@ def get_glean_aggregations(request, **kwargs):
 
     aggregation_level = kwargs["aggregationLevel"]
     # Whether to pull aggregations by version or build_id.
-    if aggregation_level == "version":
+    if aggregation_level == "version" and product == 'fenix':
         dimensions.append(Q(build_id="*"))
         counts = _get_fenix_counts(app_id, versions, ping_type, os, by_build=False)
-    elif aggregation_level == "build_id":
+    elif aggregation_level == "build_id" and product == 'fenix':
         dimensions.append(~Q(build_id="*"))
         counts = _get_fenix_counts(app_id, versions, ping_type, os, by_build=True)
+    elif aggregation_level == "build_id" and product == 'firefox':
+        dimensions.append(~Q(build_id="*"))
+        counts = _get_fog_counts(app_id, versions, ping_type, os, by_build=True)
+    elif aggregation_level == "build_id" and product == 'firefox':
+        dimensions.append(~Q(build_id="*"))
+        counts = _get_fog_counts(app_id, versions, ping_type, os, by_build=True)
```

Comment:
I think we can clean up the logic here by nesting either the `if aggregation_level ==` or `if product == ` under the other condition.

Reply:
something like this:
```
if aggregation_level == "version":
    if product == 'fenix':
        dimensions.append(Q(build_id="*"))
        counts = _get_fenix_counts(app_id, versions, ping_type, os, by_build=False)
    if product == "firefox":
        ...
elif aggregation_level == "build_id":
    if product == 'fenix':
        ...
```

although if we have one single method `_get_glean_counts` then it'd be even cleaner and we don't have to nest conditions like this.

Reply:
ok will make the change to the nested if-else